setup.py

```diff
@@ -45,7 +45,7 @@
     install_requires=[
         'dataclasses;python_version<="3.6"',
         'future',
-        'msgpack>=0.5.2',
+        'msgpack==0.6',
```

Comment:
prefer >=

Reply:
that's what got us into this mess but ok

Reply:
this just happens so regularly with python packages that it's my belief that pinning is the answer, but of course I'm happy to be overruled